manim/animation/animation.py

```diff
@@ -159,8 +164,12 @@ def interpolate_mobject(self, alpha: float) -> None:
             self.interpolate_submobject(*mobs, sub_alpha)
 
     def interpolate_submobject(
-        self, submobject: Mobject, starting_submobject: Mobject, alpha: float
-    ) -> None:
+        self,
+        submobject: Mobject,
+        starting_submobject: Mobject,
+        # target_copy: Mobject,
```

Comment:
Here is an issue that the signature of the interpolate_submobject method is different in Animation than in Transform. It needs cleaning up.. but if I add this line here uncommented, then a bunch of test fail.